ui/raidboss/data/06-ew/trial/rubicante.ts

```diff
@@ -0,0 +1,284 @@
+import Conditions from '../../../../../resources/conditions';
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { NetMatches } from '../../../../../types/net_matches';
+import { TriggerSet } from '../../../../../types/trigger';
+
+/**
+Slots:
+00 = Arena fiery or not
+01 = Inner circle
+02 = Middle ring
+03 = Outer ring
+04 = Flamespire brand indicator?
+
+00 flags:
+00020001 = Fiery
+00080004 = Not fiery
+
+01/02/03 flags:
+00020001 = Arrows rotating CW
+00080004 = Clear CW arrows
+00200010 = Arrows rotating CCW
+00400004 = Clear CCW arrows
+
+04 flags:
+00010001 = cardinals safe?
+00200020 = intercards safe?
+00080004 = clear indicator
+ */
+
+// 7CD4 Ghastly Torch during add phase *is* an aoe but is constant and small, so skipped.
+
+// TO-DO:
+//  - Rewrite Ordeal of Purgation to use Map Effects and identify directional source of cone/safe area(s)
+//  - Flamerake - identify initial direction of line cleave?
+
+export interface Data extends RaidbossData {
+  decOffset?: number;
+  dualfireTarget: boolean;
+}
+
+// First headmarker is Shattering Heat (tankbuster on MT)
+const firstHeadmarker = parseInt('0156', 16);
+const getHeadmarkerId = (data: Data, matches: NetMatches['HeadMarker']) => {
```

Comment:
Does normal mode use headmarker offsets? (In other words, is the first headmarker always literally `0156`?)  If no, can you just use the headmarker ids directly?

Mostly I ask because I don't think there's ever been a normal mode that used headmarker offsets.

Reply:
> I'd be happy to land this without the ordeals changes and have that as a follow-up.

Sounds good.  I just finished mapping (pun intended?) the rest of the Ordeals map effects - it's a little different for normal mode - and I want to think on a couple of things.  So I'll make changes/address your comments below and mark as ready, and do a follow-up PR once I've landed on an approach for the Ordeals.

Reply:
Nope, unnecessary code - we can just use the headmarkerId directly.